concepts/extension-methods/about.md

```diff
@@ -0,0 +1,22 @@
+# About
+
+[Extension methods][extension-methods] allow adding methods to existing types without creating a new derived type, recompiling, or otherwise modifying the original type.
+
+Extension methods are brought into scope at the namespace level.
+
+Extension methods are static methods, but they're called as if they were instance methods on the extended type. It achieves this by using `this` before the type, indicating the instance we put the `.` on is passed as the first parameter. For client code, there's no apparent difference between calling an extension method and the methods defined in a type.
+
+```csharp
+public static int WordCount(this string str)
```

Comment:
It might be nice to be a bit more explicit here and give the class a namespace, and then have a second code sample indicating another file that has a `using` statement for importing the extension method. That might make it a bit more explicit.